py/fiberassign/test/test_vis.py

```diff
@@ -151,11 +151,11 @@ def test_plotpos(self):
         self._load_and_plotpos(time, test_dir, suffix, simple=True)
         suffix = "{}".format(time)
         self._load_and_plotpos(time, test_dir, suffix, simple=False)
-        time = "2012-12-12T00:00:00"
-        suffix = "{}_simple".format(time)
-        self._load_and_plotpos(time, test_dir, suffix, simple=True)
-        suffix = "{}".format(time)
-        self._load_and_plotpos(time, test_dir, suffix, simple=False)
+        # time = "2012-12-12T00:00:00"
+        # suffix = "{}_simple".format(time)
+        # self._load_and_plotpos(time, test_dir, suffix, simple=True)
+        # suffix = "{}".format(time)
+        # self._load_and_plotpos(time, test_dir, suffix, simple=False)
```

Comment:
What part of this doesn't work any more?  Or is this a leftover debugging comment-out?

Reply:
Forgotten comment-out, will fix.